docs/cli.md

```diff
@@ -197,6 +197,56 @@ Usage:
       --validate=false: If true, use a schema to validate the input before sending it
 ```
 
+#### resize
+Set a new size for a resizable resource (currently only Replication Controllers)
+
+Resize also allows users to specify one or more _preconditions_ for the resize action.  The two currently supported options are current size (```--current-size```) or resource version (```--resource-version```).  If a precondition is specified, it is validated before the resize is attempted, and it is guaranteed that the precondition holds true when the resize is sent to the server.
+
+Examples:
+```sh
+  $ kubectl resize replicationcontrollers foo 3
+  resized
+```
+
+```sh
+  # will only execute if the current size is 3
+  $ kubectl resize --current-size=2 replicationcontrollers foo 3
+```
+
+Usage:
+```
+  kubectl resize [--resource-version] [--current-size] <resource> <id> <count> [flags]
```

Comment:
Done, I changed it to ```--replicas``` for now to be consistent with the existing run-container, and the notion that flags should be mechanically translateable to field names.